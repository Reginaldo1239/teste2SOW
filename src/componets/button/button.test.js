import React from 'react';
import renderer from 'react-test-renderer';

import Button from './index';


test('renders a message', () => {
    const tree1 = renderer.create(<Button  content='test' ></Button>).toJSON();
    const tree2= renderer.create(<Button  color='red' content='test' ></Button>).toJSON();
  expect(tree1).toMatchSnapshot();
  expect(tree2).toMatchSnapshot();
 
 
  })  