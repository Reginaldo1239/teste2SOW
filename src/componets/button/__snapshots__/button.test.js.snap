// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a message 1`] = `
<button
  className="ui instagram button"
  onClick={[Function]}
>
  test
</button>
`;

exports[`renders a message 2`] = `
<button
  className="ui red button"
  onClick={[Function]}
>
  test
</button>
`;
